core/include/kernel/notif.h

```diff
@@ -108,11 +108,16 @@ struct notif_driver {
  * @ops Notifier callback operations or NULL if none
  * @flags: NOTIF_IT_FLAG_* properties
  * @priv Driver private data of NULL
+ * @set_mask Copy of set_mask entry for pager optims
```

Comment:
This is a big ugly in my opinion. Better to deal with the problem in place, or accept the penalty.

Reply:
Ok, i'll accept the penalty to make thing simpler.
If one needs an optim, let's deal with it by then.